packages/block-library/src/image/edit.native.js

```diff
@@ -214,6 +216,47 @@ export default class ImageEdit extends React.Component {
 			</BottomSheet>
 		);
 
+		const getMediaOptions = () => (
+			<BottomSheet
+				isVisible={ this.state.showMediaOptions }
+				onClose={ onMediaOptionsClose }
+				hideHeader
+			>
+				<BottomSheet.Cell
+					label={ __( 'Choose from device' ) }
+					labelStyle={ { color: '#00aadc' } }
+					onPress={ onMediaUploadButtonPressed }
+				/>
+				<BottomSheet.Cell
+					label={ __( 'Take a Photo' ) }
+					labelStyle={ { color: '#00aadc' } }
+					onPress={ onMediaCaptureButtonPressed }
+				/>
+				<BottomSheet.Cell
+					label={ __( 'WordPress Media Library' ) }
+					labelStyle={ { color: '#00aadc' } }
+					onPress={ onMediaLibraryButtonPressed }
+				/>
+				<BottomSheet.Cell
+					label={ __( 'Dismiss' ) }
+					labelStyle={ { color: '#00aadc ', fontWeight: 'bold' } }
```

Comment:
There's an extra space in the color that generates a warning:

<img width="315" alt="screen shot 2019-02-05 at 9 02 20 am" src="https://user-images.githubusercontent.com/9772967/52260004-e344a900-2924-11e9-85f5-d2b57175113b.png">

<img width="391" alt="screen shot 2019-02-05 at 8 58 41 am" src="https://user-images.githubusercontent.com/9772967/52260010-e9d32080-2924-11e9-8423-41fb71491364.png">

```suggestion
					labelStyle={ { color: '#00aadc', fontWeight: 'bold' } }
```

Reply:
@etoledom thanks for the comment.
@SergioEstevao is working on WPiOS and it will be ready soon.